nss-cohort-36/keahua-arboretum-net-surfers
# Interface file structure

# Description

[Please include a summary of the change and which issue is fixed. Please also include relevant motivation and context. List any dependencies that are required for this change.]

Closes #[Enter issue #]

## Type of change

Please delete options that are not relevant.


# Testing Instructions

Make sure to commit local changes in your branch before switching to master.
1. `git checkout master`
2. `git fetch --all`
3. `git checkout interface-file-structure`
4. check that all the interfaces that we have discussed are created and in the right directories, can also check the init files that everything is imported correctly if you want to

# Checklist:
